stable/app/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.maxUnavailable }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: "{{ template "app.fullname" . }}-pdb"
+spec:
+  maxUnavailable: {{ .Values.maxUnavailable }}
+  selector:
+    matchLabels:
+      {{- include "app.selectorLabels" . | nindent 6 }}
+      app.pixxel.co.in/governed-by: "deployment"
```

Comment:
I have added an additional label to pdb such that, now it will target only deployment pods.

Reply:
should be a kubernetes label not a app.pixxel.co.in label.